### PR TITLE
Clarify that you can edit Kafka Connect replicas

### PR DESCRIPTION
**Pull request by JakeSCahill**
## Description

See https://redpandadata.slack.com/archives/C02PP8MFT4P/p1725518035071329

## Checks

- [ ] New feature
- [ ] Content gap
- [x] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc [hunk 1]
```diff
@@ -411,6 +409,22 @@ For all available settings, see the xref:reference:k-connector-helm-spec.adoc#mo
...
+=== Number of replicas
+
+You can scale the Kafka Connect Pods by modifying the `deployment.replicas` parameter in the Helm values. This parameter allows you to handle varying workloads by increasing or decreasing the number of running instances.
+
+```yml
+connectors:
+  enabled: true
+  deployment:
+    create: true
+    replicas: 3
```
Comment (RafalKorepta) [inline modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc:421]: I don't know why but connectors deployment is not created using those settings. I'm debugging it atm

Comment (RafalKorepta) [reply modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc:421]: I found few issues.

Using that change in operator deployment (Redpanda custom resource) flux is getting the following error:
```
{"level":"error","ts":"2024-09-09T10:33:46.387Z","msg":"Reconciler error","controller":"helmrelease","controllerGroup":"helm.toolkit.fluxcd.io","controllerKind":"HelmRelease","HelmRelease":{"name":"redpanda","namespace":"jake"},"namespace":"jake","name":"redpanda","reconcileID":"657fba90-b2d7-4b7d-92c5-13e3fdd3c347","error":"template: redpanda/templates/tests/test-connector-via-console.yaml:150:24: executing \"redpanda/templates/tests/test-connector-via-console.yaml\" at <include \"console.fullname\" $consoleValues>: error calling include: template: redpanda/charts/console/templates/_helpers.tpl:16:10: executing \"console.fullname\" at <include \"console.Fullname\" (dict \"a\" (list .))>: error calling include: template: redpanda/charts/console/templates/_helpers.go.tpl:22:30: executing \"console.Fullname\" at <include \"console.cleanForK8s\" (dict \"a\" (list $values.fullnameOverride))>: error calling include: template: redpanda/charts/console/templates/_helpers.go.tpl:78:48: executing \"console.cleanForK8s\" at <$s>: invalid value; expected string"}
```

The same problem using helm install

```
Error: INSTALLATION FAILED: template: redpanda/templates/tests/test-connector-via-console.yaml:150:24: executing "redpanda/templates/tests/test-connector-via-console.yaml" at <include "console.fullname" $consoleValues>: error calling include: template: redpanda/charts/console/templates/_helpers.tpl:16:10: executing "console.fullname" at <include "console.Fullname" (dict "a" (list .))>: error calling include: template: redpanda/charts/console/templates/_helpers.go.tpl:22:30: executing "console.Fullname" at <include "console.cleanForK8s" (dict "a" (list $values.fullnameOverride))>: error calling include: template: redpanda/charts/console/templates/_helpers.go.tpl:78:48: executing "console.cleanForK8s" at <$s>: invalid value; expected string
```

Give me more time and I will try to file a fix.

**Comment by JakeSCahill** [comment]
Currently blocked by a bug: https://redpandadata.atlassian.net/browse/K8S-339?atlOrigin=eyJpIjoiNmI4MjljMDk4MmYwNDE5ZTlmNTNiMGU2MmIwYTgwZGUiLCJwIjoiamlyYS1zbGFjay1pbnQifQ

**Comment by RafalKorepta** [comment]
@JakeSCahill The fix is already merged and released.

Reference:
**https://github.com/redpanda-data/helm-charts/pull/1541**
**https://github.com/redpanda-data/helm-charts/pull/1543**

**Review by RafalKorepta — APPROVED** [review]

**Review by kbatuigas — APPROVED** [review]
LGTM!

**Pull request merged**